src/functions/helpers.ts

```diff
@@ -810,10 +804,17 @@ export function linearSearch<T>(
   let closestMatchIndex = -1;
   for (let i = 0; i < numberOfValues; i++) {
     const value = normalizeValue(getValue(data, i));
+
     if (value === _target) {
       return reverseSearch ? numberOfValues - i - 1 : i;
     }
-    if (mode === "nextSmaller") {
+    if (mode === "wildcard") {
+      if (typeof value === "string" && typeof _target === "string") {
+        if (operandToRegExp(_target).test(value)) {
```

Comment:
creating a regexp can be quite slow (the regexp string is compiled to actual code).
Can it be created outside the for loop ?

Reply:
Other performance consideration: if it is easy to detect if `_target` contains regexp like characters and, if not, avoid using the regexp all together ?

Reply:
my bad